src/NumericalAlgorithms/Interpolation/SendPointsToInterpolator.hpp

```diff
@@ -28,6 +28,89 @@ struct ReceivePoints;
 
 namespace intrp {
 
+namespace InterpolationTarget_detail {
+
+/// Computes the block logical coordinates of an InterpolationTarget.
+///
+/// get_block_logical_coords is called by an Action of InterpolationTarget.
+///
+/// Currently two Actions call get_block_logical_coords:
+///   SendPointsToInterpolator (called by AddTemporalIdsToInterpolationTarget
+///                             and by FindApparentHorizon)
+///   InterpolationTargetVarsFromElement (called by DgElementArray)
```

Comment:
Same comment about doxygen format (here and below)

Reply:
:+1: